pylmod/__init__.py

```diff
@@ -20,7 +21,7 @@
         # not installed, but there is another version that *is*
         raise DistributionNotFound
 except DistributionNotFound:  # pragma: no cover
-    __version__ = 'Please install this project with setup.py'
+    __version__ = '0.1.0'  # hard coded value until we can debug RTD defect
```

Comment:
I would actually prefer you just set `VERSION = '0.1.0'` in conf.py and leave this file as is, that way doc bugs are confined to doc executable code.